ethereum/blockcreation/src/main/java/org/hyperledger/besu/ethereum/blockcreation/txselection/BlockTransactionSelector.java

```diff
@@ -505,6 +512,18 @@ private TransactionSelectionResult handleTransactionSelected(
               transactionReceiptFactory.create(
                   transaction.getType(), processingResult, cumulativeGasUsed);
 
+          maybeBlockAccessListBuilder.ifPresent(
+              blockAccessListBuilder ->
+                  processingResult
+                      .getTransactionAccessList()
+                      .ifPresent(
+                          transactionAccessList -> {
+                            if (txWorldStateUpdater
+                                instanceof StackedUpdater<?, ?> stackedUpdater) {
+                              blockAccessListBuilder.addTransactionLevelAccessList(
+                                  transactionAccessList, stackedUpdater);
```

Comment:
Why this check about the type? and if the type does not match it is fine to just ignore?

Reply:
For now we are not yet compatible with Journaled updater. This is a work that will have to do. For now, no particular problem because it's just experimental.